core/src/operations/mod.rs

```diff
@@ -552,6 +553,9 @@ pub enum Operation {
     /// $r$ and $p$, respectively. This instruction is specialized to main trace columns i.e.
     /// the values $T_i(x)$ are base field elements.
     RCombBase = OPCODE_RCOMBBASE,
+
+    /// TODO(plafer): add docs
+    Emit(u32) = OPCODE_EMIT,
```

Comment:
Similar comment to the one of the previous ones: I wonder if `Emit` should go into the "System operations" section.

Reply:
Done